src/harness/reference_models/examples/benchmark/time_dpa.py

```diff
@@ -0,0 +1,257 @@
+"""Benchmark the DPA model timing.
+
+This is a simple simulation of the DPA for purpose of estimating time to run.
+
+Usage:
+  python time_dpa.py
+
+Notes:
+  - multiprocessing facility not tested on Windows (use 1 process if issues)
+  - if warning reported on cached tiles swapping, increase the cache size.
+"""
+# TODO(sbdt): add reporting for memory usage
+from collections import namedtuple
+import multiprocessing
+import time
+
+import cartopy.crs as ccrs
```

Comment:
Maybe we need to add this and matplotlib to README.md?

Reply:
Done